src/bert/utils.py

```diff
@@ -93,7 +128,8 @@ def forward(self, sent: Dict[str, torch.LongTensor],
         if self.embeddings_mode != "only":
             # encoded_layers is a list of layer activations, each of which is
             # <float32> [batch_size, seq_len, output_dim]
-            encoded_layers, _ = self.model(ids, token_type_ids=torch.zeros_like(ids),
+            token_types = _get_seg_ids(ids, self._sep_id + 2) if is_pair_task else torch.zeros_like(ids)
```

Comment:
Should this be just `self._sep_id` (and not +2)? (Isn't it adjusted back before this, on line 113?)

Reply:
Ah nice catch!